spec/examples/expected_fall_in_must_when_type_is_first_advanced.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class ExpectedFallInMustWhenTypeIsFirstAdvanced < Actor
```

Comment:
Should this class be named Expected**Fail**InMustWhenTypeIsFirstAdvanced?

Reply:
Done: https://github.com/sunny/actor/pull/105/commits/1b0861695e361aad3c7f0443cd9148df1bf470e2